index.js

```diff
@@ -207,18 +207,15 @@
 
   'use strict';
 
-  //  deps :: Array String
-  var deps = [
-    'sanctuary-def',
-    'sanctuary-type-classes',
-    'sanctuary-type-identifiers'
-  ];
-
   /* istanbul ignore else */
   if (typeof module === 'object' && typeof module.exports === 'object') {
-    module.exports = f(require(deps[0]), require(deps[1]), require(deps[2]));
+    module.exports = f(require('sanctuary-def'),
+                       require('sanctuary-type-classes'),
+                       require('sanctuary-type-identifiers'));
   } else if (typeof define === 'function' && define.amd != null) {
-    define(deps, f);
+    define(['sanctuary-def',
+      'sanctuary-type-classes',
+      'sanctuary-type-identifiers'], f);
```

Comment:
My rule for line wrapping is that expressions at the same level semantically should be at the same level visually. I'd prefer this wrapping:

```javascript
define(['sanctuary-def',
        'sanctuary-type-classes',
        'sanctuary-type-identifiers'],
       f);
```

I seem to recall introducing `deps` to avoid the above, because `f` looked lonely. :cry:

@JAForbes, could you make this change and squash your commits while you're at it? I'll then merge this pull request and release v0.12.1. :)